mod-finance/schemas/exchange_rate.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Exchange rate schema",
+  "type": "object",
+  "properties": {
+    "from": {
+      "description": "From currency code",
+      "$ref": "../../common/schemas/currency_code.json"
+    },
+    "to": {
+      "description": "To currency code",
+      "$ref": "../../common/schemas/currency_code.json"
+    },
+    "exchangeRate": {
+      "description": "Exchange rate value",
+      "type": "string",
+      "pattern": "^\\d*\\.?\\d*$"
```

Comment:
this regex pattern not works for me. It looks like double escape symbol "\" is redundant.
Or you can try with something else like "[0-9]*\.?[0-9]+"